WTWSMS/common/dynasties/00_dynasties.txt

```diff
@@ -28130,7 +28130,7 @@
 	culture = old_slavic
 }
 1042167 = {
-	name="Atillid" 
+	name="the Hun" 
 	culture = hunnic 
```

Comment:
1. I don't know when it was used originally and in which language, but it certainly wasn't English. 
2. And also with Huns in general as I said.
3. First of all, I'm French myself. Secondly, we can't translate dynasty names, only nicknames. Thirdly, in French "le Hun" isn't used to refer to him, or at the very least I have never heard it. What you wrote was "of the Huns". Without something before the "of" it has no meaning. 
4. Attilid is used as a convention in all cases where there is no historical name, in that case the name of the main character/founder is used. 
5. Again, the Hun is an accurate nickname, nothing else, and so it will remain.